### PR TITLE
74 create component for chore frequency

### LINKED ISSUE #74: Create component for chore frequency
A reusable component for adding and editting chores
image
image

### PR DESCRIPTION
**Pull request by robinLundahl**

### CODE REVIEW

File: components/ChoreFrequency.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
```
Comment (Razew) [inline components/ChoreFrequency.tsx]: Remove this as it isn't needed
```suggestion

```

File: components/ChoreFrequency.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import { ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { Badge, Card, Text } from 'react-native-paper';
+
+const ChoreFrequency = () => {
+  const [value, setValue] = React.useState(7);
+  const [isPressed, setIsPressed] = React.useState(false);
```
Comment (Razew) [inline components/ChoreFrequency.tsx]: Same as above, .React isn't necessary
```suggestion
  const [value, setValue] = useState(7);
  const [isPressed, setIsPressed] = useState(false);
```

File: components/ChoreFrequency.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { Badge, Card, Text } from 'react-native-paper';
+
+const ChoreFrequency = () => {
```
Comment (Razew) [inline components/ChoreFrequency.tsx]: For consistency write component as a function
```suggestion
export default function ChoreFrequency() {
```

File: components/ChoreFrequency.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    width: '100%',
+  },
+  recur: {
+    flex: 1,
+  },
+  frequencyContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
```
Comment (Razew) [inline components/ChoreFrequency.tsx]: I'm not sure if this is required, seeing as it's a child of the <View> that already has alignItems: 'center' on it. I could be mistaken though, so please try it out to be on the safe side!
```suggestion

```

File: components/ChoreFrequency.tsx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    flexDirection: 'row',
+    justifyContent: 'center',
+    paddingVertical: 10,
+  },
+  numberTouchable: {
+    marginHorizontal: 5,
+  },
+});
+
+export default ChoreFrequency;
```
Comment (Razew) [inline components/ChoreFrequency.tsx]: Remove this after changing ChoreFrequency to export default function
```suggestion

```

**Review by Razew — CHANGES_REQUESTED** [review]
Looks good for the most part, just take a look at these things and we should be all good!

**Comment by Razew** [comment]
Just as a heads up for future us: the various hardcoded style colors will override the theme

File: components/ChoreFrequency.tsx (hunk 1 above)
Comment (robinLundahl) [reply components/ChoreFrequency.tsx]: fixed

File: components/ChoreFrequency.tsx (hunk 3 above)
Comment (robinLundahl) [reply components/ChoreFrequency.tsx]: fixed

File: components/ChoreFrequency.tsx (hunk 2 above)
Comment (robinLundahl) [reply components/ChoreFrequency.tsx]: fixed

File: components/ChoreFrequency.tsx (hunk 4 above)
Comment (robinLundahl) [reply components/ChoreFrequency.tsx]: fixed

File: components/ChoreFrequency.tsx (hunk 5 above)
Comment (robinLundahl) [reply components/ChoreFrequency.tsx]: fixed

**Comment by robinLundahl** [comment]
Awesome feedback! This is a great way of learning and get reminded of "slarvfel"!

**Review by Razew — APPROVED** [review]
Sweet, good job!

**Pull request merged**